lib/dfe/analytics.rb

```diff
@@ -95,8 +95,10 @@ def self.initialize!
         return
       end
 
-      unless File.exist?(Rails.root.join('config/initializers/dfe_analytics.rb'))
-        Rails.logger.info('Warning: DfE Analytics is not set up. Run `bundle exec rails generate dfe:analytics:install')
+      unless Rails.env.production? || File.exist?(Rails.root.join('config/initializers/dfe_analytics.rb'))
+        message = "Warning: DfE Analytics is not set up. Run: 'bundle exec rails generate dfe:analytics:install'"
+        Rails.logger.info(message)
+        puts message
         return
       end
```

Comment:
For my brain, `unless ||` is easier to read but happy to update to `if !`

Reply:
Randomly saw this. First time I see `unless ||` described as easier to read 😅. I personally avoid it.

Reply:
Ha, well, imo it follows the pattern with the other unless just above it, plus I think easier than if doesn't && doesn't @vassyz